packages/manager/modules/pci/src/projects/project/private-networks/private-networks.less

```diff
@@ -0,0 +1,18 @@
+.pv-data-grid {
+  .oui-datagrid-row-detail__container {
+    padding-top: 0;
+  }
```

Comment:
**nitpick:** `pt-0` bootstrap class can be used.

Reply:
grid-details is not taking the external class that's why I added inside css.